src/main/java/rx/Observable.java

```diff
@@ -5368,6 +5368,27 @@ public final Boolean call(T t) {
     }
     
     /**
+     * Instructs an Observable that is emitting items faster than its observer can consume them to 
+     * hold onto the latest value and emit that on request.
+     * <p>
+     * Its behavior is logically equivalent to toBlocking().latest() with the exception that
+     * the downstream is not blocking while requesting more values.
+     * <p>
+     * Note that if the upstream Observable does support backpressure, this operator ignores that capability
+     * and doesn't propagate any backpressure requests from downstream.
+     * <p>
+     * Note that due to the nature of how backpressure requests are propagated through subscribeOn/observeOn,
```

Comment:
This sentence isn't clear to me. I imagine it has something to do with how `request(n)` generally will happen in batches so `onNext` emissions will happen in bursts and drop everything but the latest in between in each burst?